packages/ses/test/test-enable-default-overrides-default.js

```diff
@@ -0,0 +1,50 @@
+import '../lockdown.js';
+import test from 'ava';
+
+lockdown({
+  // errorTaming: 'unsafe',
+});
+
+// See https://github.com/endojs/endo/issues/616#issuecomment-800733101
+
+test('enable default overrides of Uint8Array', t => {
+  t.notThrows(() => {
+    function Buffer(_arg, _encodingOrOffset, _length) {}
+    Object.setPrototypeOf(Buffer.prototype, Uint8Array.prototype);
+    Object.setPrototypeOf(Buffer, Uint8Array);
+    Buffer.prototype.toLocaleString = Buffer.prototype.toString;
+  });
+});
+
+test('enable default overrides of Uint8Array in evaluation', t => {
+  const c = new Compartment();
+  t.notThrows(() =>
+    c.evaluate(
+      `(${function foo() {
```

Comment:
The compartment and evaluation were not contributing to the test so I preceded them with simplified tests without those. The had identical behaviors under development except that the evaluate prevented errors from showing meaningful line numbers.